test/rofl/common/crofbase/crofbasetest.cpp

```diff
@@ -83,21 +79,21 @@ void cdatapath::test_start() {
 }
 
 void ccontroller::handle_dpt_open(rofl::crofdpt &dpt) {
-  std::cerr << ">>> XXX dpt connected: " << std::endl;
+  LOG(INFO) << ">>> XXX dpt connected: " << std::endl;
```

Comment:
not sure what the original intention was, but it could be done like this, will add it in a separate PR.